package.json

```diff
@@ -47,28 +52,34 @@
     "stylint": "1.5.9"
   },
   "dependencies": {
-    "@atlaskit/editor-core": "134.0.1",
+    "@atlaskit/editor-core": "acezard/editor-core-fork#134.0.1",
+    "@atlaskit/icon": "21.1.0",
     "@atlaskit/media-core": "32.1.2",
-    "@atlaskit/media-editor": "38.1.4",
+    "@atlaskit/media-editor": "^38.1.0",
     "@atlaskit/smart-card": "14.6.1",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-typescript": "^7.14.5",
+    "@babel/runtime": "^7.0.0",
     "@material-ui/core": "4",
     "@material-ui/styles": "4.10.0",
+    "classnames": "^2.2.5",
     "cozy-bar": "7.7.8",
     "cozy-client": "23.1.2",
     "cozy-device-helper": "1.8.0",
     "cozy-doctypes": "1.70.0",
     "cozy-flags": "2.1.0",
     "cozy-realtime": "3.13.0",
-    "cozy-scripts": "5.1.1",
+    "cozy-scripts": "acezard/cozy-scripts-ts",
     "cozy-sharing": "3.3.1",
     "cozy-ui": "45.1.0",
     "eslint-config-cozy-app": "1.3.3",
-    "lodash": "4.17.21",
-    "prop-types": "^15.7.2",
+    "lodash": "^4.17.15",
+    "prop-types": "^15.5.10",
     "react": "16.12.0",
     "react-dom": "16.12.0",
     "react-intl": "2.9.0",
     "react-router-dom": "5.1.2",
+    "react-select": "^4.3.1",
```

Comment:
Why is it needed?

Reply:
If I remember correctly there were lots of conflicts from different libs having differents react-select version, and it blew the runtime up. It was in the beginning of dev though so maybe it is not needed anymore I can check into it

Reply:
Fixed

Reply:
How? Why is it needed in our case?